### PR TITLE
Add step 0

### PR DESCRIPTION
**Pull request by Ara225**
## What does this change?

This adds ITD step zero which previously only existed in the dashboard. In addition, it makes some edits to step zero to conform with the requirements of #1057  

### Common Content?

No

### Common Theme?

No

Issue number: #1057 #958 

### Org Content?

No

## Checklist

- [x ] I have read the [contributing guidelines](CONTRIBUTING.MD)
- [ x] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [ x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [ x] My changes follow our [Style Guide](https://curriculum.codeyourfuture.io/guides/code-style-guide)

## Who needs to know about this?
@kfklein15

### CODE REVIEW

**Review by SallyMcGrath — COMMENTED** [review]
simplify language, fix paths

File: org-cyf-itd/content/steps/zero/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
++++
+title = 'Zero'
+description = 'Join the CodeYourFuture Intro to Digital Slack'
+layout = 'prep'
+emoji= '🗨'
+menu_level = ['module']
+menu=[ 'steps']
+weight = 1
```
Comment (SallyMcGrath) [inline org-cyf-itd/content/steps/zero/index.md:8]:
```suggestion
weight = 0
```

File: org-cyf-itd/content/steps/zero/index.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ layout = 'prep'
 emoji= '🗨'
 menu_level = ['module']
 menu=[ 'steps']
-weight = 1
+weight = 0
```
Comment (SallyMcGrath) [inline org-cyf-itd/content/steps/zero/index.md:8]: You will have to run through all the steps and update the weight so this should be 1, 1 is 2, 3 is 4... etc.

The step 0 addition was a curveball!

**Review by SallyMcGrath — COMMENTED** [review]
@Ara225 I've updated the paths directly because it's faster than explaining. I've left the weight update for you to do - and put the deploy through so you can see the effect.

**Comment by Ara225** [comment]
Thank you 😊

**Comment by Ara225** [comment]
Done, makes sense

**Review by SallyMcGrath — APPROVED** [review]
lgtm

**Pull request merged**